at-hoangtran/Week3/Day5/Exercise1-Day5.rb

```diff
@@ -0,0 +1,35 @@
+class Student
+  attr_accessor :first_name, :last_name, :list_of_courses
+  def initialize
+    @list_of_courses = []
+  end
+
+  def first_name
+    puts "#{@first_name}"
+  end
+  def last_name
+    puts "#{@last_name}"
+  end
+  def full_name
+    puts "#{@first_name}" + " #{@last_name}"
+  end
+  def add_course(course_name) 
+    unless @list_of_courses.include?(course_name)
+      @list_of_courses.push(course_name) 
+    else
+      puts false
+    end
+  end
+  def remove_course(course_name)
+    @list_of_courses.delete(course_name) if @list_of_courses.include?(course_name)
+  end
+end
+student = Student.new
+student.list_of_courses = ["math", "history"]
+student.first_name = 'Tran'
+student.last_name = 'Hoang'
+student.full_name
+student.first_name
+student.add_course("physical")
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/atruby-team/internship-18-summer/blob/074932088ed5cad5e69983d076619cff8875320e/at-hoangtran/Week3/Day5/Exercise1-Day5.rb#L33"></span>Style/StringLiterals: Prefer single-quoted strings when you don’t need string interpolation or special symbols. (https://github.com/rubocop-hq/ruby-style-guide#consistent-string-literals)</td>
    </tr>
  </tbody>
</table>